### PR TITLE
Iteration 18

### PR DESCRIPTION
**Pull request by franchu7**
**Start open elevation service in grpc mode**

### CODE REVIEW

**Review by liu-hanli — APPROVED** [review]
OK

File: openelevationservice/server/api/querybuilder.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 from openelevationservice import SETTINGS
 from openelevationservice.server.utils.logger import get_logger
-from openelevationservice.server.db_import.models import db, Cgiar
+#from openelevationservice.server.db_import.models import db, Cgiar
+from openelevationservice.server.grpc.db_grpc import db, Cgiar
 # from openelevationservice.server.utils.custom_func import ST_SnapToGrid
```
[the remark this answers is not in the document]
Comment (franchu7) [reply openelevationservice/server/api/querybuilder.py:7]: DONE

File: README.rst [hunk 2]
```diff
@@ -152,7 +149,9 @@ When your environment is set up, you can run the import process and start the se
    flask importdata
 
    # Start the server
-   flask run
+   flask --app manage run # HTTP mode
+   # or
+   flask --app manage grpc # GRPC mode
```
[the remark this answers is not in the document]
Comment (franchu7) [reply README.rst:154]: DONE

File: requirements.txt [hunk 3]
```diff
@@ -16,4 +16,7 @@ gevent>=23.9.0
 requests>=2.20.0
 nose>=1.3.0
 numpy>=1.25.0
-psycopg2-binary>=2.8.4
\ No newline at end of file
+psycopg2-binary>=2.8.4
+grpcio-tools>=1.54.0
+setuptools>=65.5.1
+grpcio-reflection>=1.54.0
```
[the remark this answers is not in the document]
Comment (franchu7) [reply requirements.txt:22]: I forgot to reply to the comment 😅. DONE

**Pull request merged**